File: Serene3/Serene3.Web/Modules/com/review2/review2Columns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace Serene3.com.Columns;

[ColumnsScript("com.review2")]
[BasedOnRow(typeof(review2Row), CheckNames = true)]
public class review2Columns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    public string ProductName { get; set; }
    [EditLink]
    public string Name { get; set; }
    public string Email { get; set; }
    public string Subject { get; set; }
    public string Description { get; set; }
    public int Rating { get; set; }
    public DateTime CreatedAt { get; set; }
}

File: Serene3/Serene3.Web/Modules/ServerTypes/com/review2Columns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { review2Row } from "./review2Row";

export interface review2Columns {
    Id: Column<review2Row>;
    ProductName: Column<review2Row>;
    Name: Column<review2Row>;
    Email: Column<review2Row>;
    Subject: Column<review2Row>;
    Description: Column<review2Row>;
    Rating: Column<review2Row>;
    CreatedAt: Column<review2Row>;
}

export class review2Columns extends ColumnsBase<review2Row> {
    static readonly columnsKey = 'com.review2';
    static readonly Fields = fieldsProxy<review2Columns>();
}